README.md

```diff
@@ -1,11 +1,26 @@
 # Bitcoin Armory Website
+## Overview
+This is the website for the community-developed version of [Bitcoin Armory](https://btcarmory.com/), with lead development by [goatpig](https://github.com/goatpig/BitcoinArmory). This site uses the following technologies.
 
-This is the website for the Open Source version of Bitcoin Armory being developed by goatpig. This site uses the [jekyll-now theme](https://github.com/barryclark/jekyll-now)
+- [Ruby](http://www.ruby-lang.org/en/)
+- [Jekyll](https://jekyllrb.com/)
+- [Bundler](https://bundler.io/)
+- [github-pages](https://pages.github.com/)
+- [The *jekyll-now* theme](https://github.com/barryclark/jekyll-now)
+
+While goatpig, the lead Armory maintainer, controls the web site, users are encouraged to contribute updates to the web site (e.g., tips for novice users). goatpig maintains strict control of the Armory repo but is more of a "sanity checker" for the web site, simply making sure the [pull requests](https://github.com/goatpig/BitcoinArmory/tree/gh-pages) are sensible.
 
 ## Building the website locally
+The site relies on Jekyll in order to be developed locally. Ruby development can be tricky, especially cross-platform development. The following steps have been tested on Linux (Ubuntu) and macOS, and should work on Windows 10.
 
-Clone the repo. In terminal, type `gem install bundle github-pages`. Then run the server with
+- *(OPTIONAL)* Clone the repo. This is necessary only if you wish to submit a PR and attempt to update the web page. (NOTE: When making a commit to your repo, you may receive an email from GitHub, stating that you don't control btcarmory.com. This issue is being investigated but can be ignored for now.)
```

Comment:
> NOTE: When making a commit to your repo, you may receive an email from GitHub, stating that you don't control btcarmory.com. This issue is being investigated but can be ignored for now.

It's because the site is hosted using github pages and the only way that you can use a custom domain with github pages is to have the CNAME file which has the custom domain in it. Unfortunately this means that github will attempt to assign every repo which has that CNAME file in it the domain that's listed in the CNAME (i.e. it will attempt to list forks). But the domain is already taken by one repo (this) and the control of the site will remain there. There really is no way around that problem I think.

Reply:
Makes sense. I'll take a minute to look into this but I'm guessing I'll just have to do a fixup and let people know the emails are inevitable.